### DISCUSSION TITLE
Allow server owner to specify own map tile server

### DISCUSSION THREAD

**Discussion by marsmathis** [Freika/dawarich#429] [Ideas]
I’d like to be able to use my own self-hosted map tile server (in my case that’d be versatiles). This should not be too hard to implement (after all, you pull the tiles from existing tile servers already with the given options) and this would allow people to specify exactly how they want their map to be. You may want to look at Traccar, they allow you to specify a tile server through a custom URL.

Edit: Please, if you do this, also make sure that vector tiles are supported.

**Comment by Freika** [comment]
Done in 0.24.1

**Reply by marsmathis** [reply]
You’re an absolute legend.
